requirements.txt

```diff
@@ -13,3 +13,4 @@ v3io-frames>=0.10.2,<1
 # which breaks v3io-frames
 protobuf>=3.20.1,!=3.20.2,<4
 v3iofs~=0.1.9
+xxhash>=1
```

Comment:
i noticed that we do "from kafka import" but in mlrun we install kafka-python (and indeed it did not work for me with pip install kafka). so we should probably change to import kafka-python..

Reply:
There is no `import kafka-python`. kafka-python is the name of library, kafka is the name of the main package in that library.